app/models/user.rb

```diff
@@ -1,7 +1,9 @@
 class User < ApplicationRecord
   has_many :items
   has_many :orders
-  
-  validates_presence_of :password_digest, :role, :active, :name, :address, :city, :state, :zip
+
+  validates_presence_of :password_digest, :name, :address, :city, :state, :zip
```

Comment:
@VinceCarollo What was the validation you were using for `:role` ?

Reply:
validates_inclusion_of :active, in: [true, false]

Reply:
validates_inclusion_of :active, in: [true, false]